examples/replace-component/src/components/panel-toggle.js

```diff
@@ -21,7 +21,7 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import {PanelToggleFactory, Button, Icons, withState} from 'kepler.gl/components';
+import {PanelTabFactory, PanelToggleFactory, Button, Icons, withState} from 'kepler.gl/components';
```

Comment:
Directly implement CustomPanelToggleFactory using PanelToggleFactory.deps is a safer path to prevent future breakage when PanelToggleFactory.deps changes
```js
import React from 'react';
import styled from 'styled-components';

const CustomPanelToggleFactory = (...deps) => {
  const PanelToggle = PanelToggleFactory(...deps);

  const PanelToggleWrapper = props => (
    <StyledPanelToggleWrapper>
      <PanelToggle {...props} />
      <ButtonWrapper>
        <Button onClick={() => props.onClickSaveConfig(props.mapState)} width="120px">
          <Icons.Files height="12px" />
          Save Config
        </Button>
      </ButtonWrapper>
    </StyledPanelToggleWrapper>
  );

  return withState(
    // lenses
    [visStateLens],
    // mapStateToProps
    state => ({mapState: state.keplerGl.map1}),
    {
      onClickSaveConfig: setMapConfig
    }
  )(PanelToggleWrapper);
};

CustomPanelToggleFactory.deps = PanelToggleFactory.deps;

export default CustomPanelToggleFactory;
```